source/fundamentals/connection/tls.txt

```diff
@@ -0,0 +1,296 @@
+.. _node-connect-tls:
+
+==========================
+Enable TLS on a Connection
+==========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to connect to MongoDB instances with
+the TLS security protocol.
+
+To configure your connection to use TLS, enable
+the TLS option and provide your certificates for validation.
+
+.. tip::
+   
+   To learn more about TLS, see the Wikipedia entry on
+   :wikipedia:`Transport Layer Security <Transport_Layer_Security>`.
+
+.. _node-enable-tls:
+
+Enable TLS
+----------
+
+You can enable TLS on a connection to your MongoDB instance
```

Comment:
Might be worth calling out that if you are an `srv` record user, TLS is enabled by default.